src/reports.cpp

```diff
@@ -961,20 +961,20 @@ static config unit_weapons(reports::context & rc, const unit *attacker, const ma
 
 static config unit_weapons(reports::context & rc, const unit *u)
 {
-	if (!u || u->attacks().empty()) return config();
-	map_location displayed_unit_hex = rc.screen().displayed_unit_hex();
-	config res;
-
-	//TODO enable after the string frezze is lifted
-	//const std::string attack_headline =
-	//		( u->attacks().size() > 1 ) ? N_("Attacks") : N_("Attack");
+	config res = config();
+	if ((u != NULL) && (!u->attacks().empty())) {
+		map_location displayed_unit_hex = rc.screen().displayed_unit_hex();
 
-	//add_text(res,  /*span_color(font::weapon_details_color)
-	//		+*/ attack_headline /*+ "</span>\n"*/ + '\n', "");
+		//TODO enable after the string frezze is lifted
```

Comment:
Could you address this TODO while you're here? We don't have a string freeze in effect at the moment.